ci-operator/step-registry/ipi/conf/kubevirt/install/ipi-conf-kubevirt-install-commands.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+if [[ -z "$OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE" ]]; then
+  echo "OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE is an empty string, exiting"
+  exit 1
+fi
+
+echo "Installing from release ${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE}"
+export SSH_PRIV_KEY_PATH=${CLUSTER_PROFILE_DIR}/ssh-privatekey
+export PULL_SECRET_PATH=${CLUSTER_PROFILE_DIR}/pull-secret
+export OPENSHIFT_INSTALL_INVOKER=openshift-internal-ci/${JOB_NAME}/${BUILD_ID}
+export HOME=/tmp
+
+case "${CLUSTER_TYPE}" in
+aws) export AWS_SHARED_CREDENTIALS_FILE=${CLUSTER_PROFILE_DIR}/.awscred;;
+azure4) export AZURE_AUTH_LOCATION=${CLUSTER_PROFILE_DIR}/osServicePrincipal.json;;
+gcp) export GOOGLE_CLOUD_KEYFILE_JSON=${CLUSTER_PROFILE_DIR}/gce.json;;
+kubevirt) export KUBEVIRT_KUBECONFIG=${HOME}/.kube/config;;
+vsphere) ;;
+openstack) export OS_CLIENT_CONFIG_FILE=${CLUSTER_PROFILE_DIR}/clouds.yaml ;;
+openstack-vexxhost) export OS_CLIENT_CONFIG_FILE=${CLUSTER_PROFILE_DIR}/clouds.yaml ;;
+*) echo >&2 "Unsupported cluster type '${CLUSTER_TYPE}'"
+esac
```

Comment:
Why does a file names ipi-conf-**kubevirt**-install-commands.sh should handle cases of other CLUSTER_TYPES ? I would expect to get the Unsupported cluster type for all types except for kubevirt.

Reply:
This is a side branch for testing our usecase
It's only for testing our image - I must duplicate the commands.sh file but use env variable with our image